wizeline/project-lab
# Feat - 78 - Ability to update proposal's owner on edit page.

#### What does this PR do?

It adds the ability to update proposal's owner on edit page.

#### Where should the reviewer start?

Take a look at these files: 
- app/projects/mutations/updateProject.ts
- app/projects/queries/getProject.ts
- app/projects/validations.ts

#### How should this be manually tested?

1. Click on any proposal from the home page.
2. Click on the edit icon.
3. Select from the Owner dropdown a different owner from the list.
4. Finally click on update button.
5. And then Click once again the edit icon and you should see the new owner you just chose.

#### What are the relevant tickets?

#78
